docs/docs/sourcing-from-buttercms.md

```diff
@@ -49,22 +49,25 @@ Make sure to add your **API_TOKEN** from your dashboard. In this guide you will
 
 ```javascript:title=gatsby-config.js
 module.exports = {
-{
+  plugins: [
+    {
       resolve: `gatsby-source-buttercms`,
       options: {
-        authToken: `your_auth_token`,
-        // Optional. Returns values for the supplied content field keys.
+        authToken: `<API_TOKEN>`,
+        // Optional array of Collection key
         contentFields: {
-          keys: [`faq_items`, `faq_headline`],
+          keys: [`collection_key`],
```

Comment:
I say "we're making this more generic" in my previous comment because I'm assuming "`faq_items`, `faq_headline`" might still be valid keys as well as the following `homepage` page and `customer_case_study` page type. They're referenced specifically in the preceding sentence (line 48), so I think that should be updated as well.